### PR TITLE
feat(components/tokens): add tokens to components that had none

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Add tokens to components that had none

### CODE REVIEW

**Review by mohamedMok — APPROVED** [review]

File: packages/styles/components/_c.hero.scss [hunk 1]
```diff
@@ -68,7 +68,7 @@
   }
 
   &__content {
-    color: $color-grey-000;
+    color: $color-font-lightest;
```
Comment (caqueste) [inline packages/styles/components/_c.hero.scss:71]: Replace -lightest to -revert

File: packages/styles/components/_c.phone-number.scss [hunk 2]
```diff
@@ -14,8 +14,8 @@
 
     display: flex;
     align-items: center;
-    color: $color-grey-900;
-    border: get-border("s") solid $color-grey-600;
+    color: $color-font-darker;
```
Comment (caqueste) [inline packages/styles/components/_c.phone-number.scss:17]: Replace -darker to -primary

File: packages/styles/components/_c.stepper.scss [hunk 3]
```diff
@@ -57,36 +57,36 @@
...
-        background-color: $color-primary-01-400;
+        background-color: $color-stepper-item-current-background;
       }
 
       #{$parent} {
         &__indicator {
-          background-color: $color-primary-01-400;
-          color: $color-grey-000;
+          background-color: $color-stepper-item-current-background;
+          color: $color-font-lightest;
```
Comment (caqueste) [inline packages/styles/components/_c.stepper.scss:77]: Replace -lightest to -invert

File: packages/styles/components/_c.stepper.scss [hunk 4]
```diff
@@ -57,36 +57,36 @@
...
         }
 
         &__label {
-          color: $color-grey-999;
+          color: $color-font-darkest;
```
Comment (caqueste) [inline packages/styles/components/_c.stepper.scss:81]: Replace -darktest to -primary

File: packages/styles/components/_c.tooltip.scss [hunk 5]
```diff
@@ -19,17 +19,17 @@
   &__content {
     @include set-font-scale("04", "m");
 
-    background-color: $color-grey-700;
+    background-color: $color-tooltip-background;
     border-radius: get-border-radius("m");
     box-sizing: border-box;
-    color: $color-grey-000;
+    color: $color-font-lightest;
```
Comment (caqueste) [inline packages/styles/components/_c.tooltip.scss:25]: Replace -lightest to -invert

**Review by caqueste — APPROVED** [review]

File: packages/styles/components/_c.hero.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiloyi) [reply packages/styles/components/_c.hero.scss:71]: @caqueste as seen together, this point will be fixed in a new PR on the `@next` version

File: packages/styles/components/_c.breadcrumb.scss [hunk 6]
```diff
@@ -104,23 +109,19 @@
...
         &:not(:first-child) {
-          background-image:
-            url(
-              inline-icons(
-                "arrow-right-16",
-                $color-grey-000
-              ));
+          background-image: url(inline-icons(
+            "arrow-right-16",
+            $color-breadcrumb-dark-arrow
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply packages/styles/components/_c.breadcrumb.scss]: fixed

File: packages/styles/components/_c.breadcrumb.scss [hunk 7]
```diff
@@ -104,23 +109,19 @@
...
           @media screen and (max-width: ($screen-l - 1)) {
-            background-image:
-              url(
-                inline-icons(
-                  "arrow-left-16",
-                  $color-grey-000
-                ));
+            background-image: url(inline-icons(
+              "arrow-left-16",
+              $color-breadcrumb-dark-arrow
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply packages/styles/components/_c.breadcrumb.scss]: Fixed

File: packages/styles/components/_c.option-button.scss [hunk 8]
```diff
@@ -10,12 +10,12 @@
     @include set-hidden-input;
 
     &:hover + #{$parent}__label {
-      box-shadow: 0 0 0 2px $color-grey-800;
+      box-shadow: 0 0 0 2px $color-option-button-hover-label-shadow;
     }
 
     &:checked + #{$parent}__label {
-      border-color: $color-primary-01-500;
-      box-shadow: 0 0 0 $mu025 $color-success-100;
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply packages/styles/components/_c.option-button.scss:18]: Fixed

File: packages/styles/components/_c.option-button.scss [hunk 9]
```diff
@@ -24,8 +24,8 @@
 
     &:disabled + #{$parent}__label {
       cursor: not-allowed;
-      background-color: $color-grey-200;
-      color: $color-grey-600;
+      background-color: $color-option-button-disabled-label-background;
+      color: $color-font-light;
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply packages/styles/components/_c.option-button.scss:28]: @caqueste as seen together, this point will be fixed in a new PR on the `@next` version

File: packages/styles/components/_c.option-card.scss [hunk 10]
```diff
@@ -32,12 +32,12 @@
       }
 
       &:hover + #{$parent}__label {
-        box-shadow: 0 0 0 2px $color-grey-800;
+        box-shadow: 0 0 0 2px $color-option-card-hover-label-shadow;
       }
 
       &:checked + #{$parent}__label {
-        border-color: $color-primary-01-500;
-        box-shadow: 0 0 0 $mu025 $color-success-100;
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply packages/styles/components/_c.option-card.scss:40]: Fixed

File: packages/styles/components/_c.hero.scss (hunk 1 above)
Comment (tiloyi) [reply packages/styles/components/_c.hero.scss:71]: @caqueste as seen together, this point will be fixed in a new PR on the `@next` version

File: packages/styles/components/_c.phone-number.scss (hunk 2 above)
Comment (tiloyi) [reply packages/styles/components/_c.phone-number.scss:17]: @caqueste as seen together, this point will be fixed in a new PR on the `@next` version

File: packages/styles/components/_c.stepper.scss (hunk 3 above)
Comment (tiloyi) [reply packages/styles/components/_c.stepper.scss:77]: @caqueste as seen together, this point will be fixed in a new PR on the `@next` version

File: packages/styles/components/_c.stepper.scss (hunk 4 above)
Comment (tiloyi) [reply packages/styles/components/_c.stepper.scss:81]: @caqueste as seen together, this point will be fixed in a new PR on the `@next` version

File: packages/styles/components/_c.tooltip.scss (hunk 5 above)
Comment (tiloyi) [reply packages/styles/components/_c.tooltip.scss:25]: @caqueste as seen together, this point will be fixed in a new PR on the `@next` version

**Pull request merged**